### ISSUE TITLE
The windows wheel on PyPI doesn't match any platform

### ISSUE THREAD

**Issue by frostming** [sdispater/pendulum#456]
- [x] I am on the [latest](https://github.com/sdispater/pendulum/releases/latest) Pendulum version.
- [x] I have searched the [issues](https://github.com/sdispater/pendulum/issues) of this repo and believe that this is not a duplicate.

- **OS version and name**: `Python 3.8.2 (tags/v3.8.2:7b3ab59, Feb 25 2020, 23:03:10) [MSC v.1916 64 bit (AMD64)]`
- **Pendulum version**: 

## Issue
```
pip install https://files.pythonhosted.org/packages/92/df/e93d3923a92a56d27ebc14835b066d324153d238c0f421c6723230bf71a2/pendulum-2.1.0-cp38-cp38m-win_amd64.whl
ERROR: pendulum-2.1.0-cp38-cp38m-win_amd64.whl is not a supported wheel on this platform.
```
On Python 3.8 the tag `m` doesn't exist any more, so the wheel cannot be installed on my Windows x64 Python.

**Comment by frostming** [comment]
The full list of supported tags on my machine:
```
cp38-cp38-win_amd64
cp38-abi3-win_amd64
cp38-none-win_amd64
cp37-abi3-win_amd64
cp36-abi3-win_amd64
cp35-abi3-win_amd64
cp34-abi3-win_amd64
cp33-abi3-win_amd64
cp32-abi3-win_amd64
py38-none-win_amd64
py3-none-win_amd64
py37-none-win_amd64
py36-none-win_amd64
py35-none-win_amd64
py34-none-win_amd64
py33-none-win_amd64
py32-none-win_amd64
py31-none-win_amd64
py30-none-win_amd64
cp38-none-any
py38-none-any
py3-none-any
py37-none-any
py36-none-any
py35-none-any
py34-none-any
py33-none-any
py32-none-any
py31-none-any
py30-none-any
```

**Comment by nirvana-msu** [comment]
I can confirm I am facing the exact same issue with a private package built with Poetry 1.0.5.

There is an open pull request for Poetry to fix the issue in [python-poetry/poetry/pull/2121](https://github.com/python-poetry/poetry/pull/2121)

**Comment by sdispater** [comment]
This should be fixed now.
